src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -283,10 +283,8 @@ export class RetireJsEngine implements RuleEngine {
 				// Once we've got a path, ew can copy the original target file to the child directory, thereby preserving
 				// path uniqueness. No race condition exists, so no need for an `await`.
 				fileCopyPromises.push(new Promise((res, rej) => {
-					// NOTE: We recognize that there are potential performance concerns for copying a possibly-large number
-					// of possibly large files. If those issues become a problem, we should consider alternatives such as
-					// symlink() (if RetireJS supports it) or even a different implementation.
-					fs.copyFile(p, path.join(pathAlias, path.basename(p)), (err) => {
+					// Use symlink() instead of copyFile() for performance reasons.
+					fs.symlink(p, path.join(pathAlias, path.basename(p)), (err) => {
```

Comment:
Minor nitpick - we've been using FileHandler module for calling all `fs` related methods so that we can mock them in tests easily. Have you considered creating a new method for symlink?

Reply:
Good call. Done.